recipes/pyramid_log/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyramid_log" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 165ce9460548b09061db6023b5079eb6290d9490a2b4cf28946e3515185d0a52
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyramid
+    # actually required by pyramid on 2.7 -- but not there in the main package
+    - repoze.lru
+
+test:
+  imports:
+    - pyramid_log
+
+about:
+  home: https://pypi.org/project/pyramid-log/
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
```

Comment:
Looks like the license file is already package in the pypi tarball:
```
pyramid_log-0.2.1/
pyramid_log-0.2.1/MANIFEST.in
pyramid_log-0.2.1/setup.py
pyramid_log-0.2.1/README.rst
pyramid_log-0.2.1/LICENSE
```
so, please remove the file LICENSE.txt, and rename the license_file to LICENSE.

Reply:
good catch, thanks!